doc/content/resources/multiapps_chps/chp_3_applications.md

```diff
@@ -0,0 +1,123 @@
+
+Localized physics such as performance of coolant channels, heat pipes, and fuel can also be handled by grandchild applications. These local applications receive temperature and neutronics information from parent and child applications and perform their specific computation in their specific domain. In some cases, the local physics results are important to reactor thermal and neutronics, such as fuel volumetric swelling strain and coolant temperature, which will need to be transferred back to the parent and child applications for Picard iteration. In other cases, the local physics results do not affect reactor thermal and neutronics physics, such as fuel-cladding interaction, and can be loosely coupled without iteration.
+
+### MultiApp Reactor Examples on VTB
+
+The examples available on the Virtual Test Bed site adopt a number of different coupling approaches based on the specific reactor type and phenomena of focused interest. We briefly summarize some examples to show readers the flexibility in using MultiApps (see [vtb_examples]).
+
+!media media/resources/vtb_examples.png
+      style=display: block;margin-left:auto;margin-right:auto;width:100%;
+      id=vtb_examples
+      caption=Hierarchies of MultiApps for VTB examples.
+
+#### VTB-MSFR
+
+The Molten Salt Fast Reactor ([MSFR](msfr/index.md)) example adopts a two-level MultiApps structure with Griffin as the parent app and Pronghorn as the child app. The parent app governs neutronics simulation, providing power distribution to the child app. The child app handles fluid dynamics simulation of molten salt fuels, providing temperature information as feedback to the parent app. MSFR is a special class of reactors as the molten salt is both fuel and coolant. As the molten salt flows out of the core region to transfer heat, the delayed neutron information is also calculated to transferred back to Griffin.
+
+#### VTB-HTGR
+
+In the High-Temperature Gas-cooled Reactor ([HTGR](htgr/index.md)) category, a Pebble-Bed Microreactor ([PB-MR](pbmr/index.md))) model is available. Griffin is the parent application governing neutronics simulations, and MOOSE Heat Conduction and Thermal Hydraulics modules child applications calculate the heat transfer in solid (matrix + fuel) and fluid (coolant) regions, respectively. A special approach used in this example is the use of a CentroidMultiApp as the grandchild app to calculate the representative temperature profile within single TRISO particles using Heat Conduction Modules. This is an example of using MultiApps to handle similar physical phenomena (thermal here) at different space scales.
```

Comment:
```suggestion
A Pebble-Bed Microreactor model is available on the VTB using a three-level MultiApps structure. Griffin is the parent application governing neutronics simulations, while Pronghorn solves a homogenized porous media flow problem. A CentroidMultiApp is then used to calculate the representative temperature profile within pebbles and TRISOs. This is an example of using MultiApps to handle similar physical phenomena (thermal heat transfer here) at different space scales. 
```

both child input files use Pronghorn not the modules. There's a lot of modules code in them but you cant run them without prognhorn because of the matprops